packages/next-ui/LayoutParts/DesktopNavBarItem.tsx

```diff
@@ -42,7 +42,7 @@ export function DesktopNavItem(props: DesktopNavItemLinkProps | DesktopNavItemBu
 
   const { href, children, sx = [], ...linkProps } = props
 
-  const active = router.asPath.startsWith((href ?? '').toString())
+  const active = router.asPath === (href ?? '').toString()
```

Comment:
The idea was that if a child is selected that the parent would be selected as well, so that breaks with this, any suggestions?

Reply:
@paales see my latest changes 🙂 

Now it activates the parent and only the menu item that exactly matches the `router.asPath`.

https://github.com/graphcommerce-org/graphcommerce/pull/2369/files